source/features/clean-repo-tabs.tsx

```diff
@@ -5,9 +5,9 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import fetchDom from '../helpers/fetch-dom';
+import * as api from '../github-helpers/api';
 import getTabCount from '../github-helpers/get-tab-count';
 import looseParseInt from '../helpers/loose-parse-int';
-import {getWorkflows} from './scheduled-and-manual-workflow-indicators';
```

Comment:
The raw workflow files were also used here, but since only their count matters I've uncoupled the two API calls completely.